Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.9.0-alpine3.12
+
+RUN pip install --upgrade google-api-python-client google-auth-httplib2 google-auth-oauthlib
+
+COPY ./requirements.txt ./requirements.txt
+
+RUN pip install -r requirements.txt
+
+COPY ./config.json ./config.json
+COPY ./export.json ./export.json
+COPY ./credentials.json ./credentials.json
```

Comment:
This file does not exists and has no sense to be included in the image as it's a runtime file

Reply:
done